### ISSUE TITLE
Uncaught Error: Cannot find module 'react/lib/Object.assign'

### ISSUE THREAD

**Issue by acomito** [WickyNilliams/react-clipboard#13]
Using this in a meteor app and it seems to have trouble finding react

```

  "dependencies": {
    "antd": "^2.9.3",
    "apollo-client": "^1.0.0-rc.3",
    "apollo-errors": "^1.4.0",
    "apollo-resolvers": "^1.0.2",
    "autoprefixer": "^6.7.7",
    "babel-runtime": "^6.23.0",
    "bcrypt": "^0.8.7",
    "body-parser": "^1.17.1",
    "classnames": "^2.2.5",
    "cors": "^2.8.3",
    "express": "^4.15.2",
    "graphql": "^0.9.5",
    "graphql-loader": "^1.2.1",
    "graphql-server-express": "^0.6.0",
    "graphql-subscriptions": "^0.3.1",
    "graphql-tag": "^1.3.1",
    "graphql-tools": "^0.11.0",
    "indexof": "0.0.1",
    "invariant": "^2.2.1",
    "lodash": "^4.17.4",
    "meteor-apollo-accounts": "^2.0.1",
    "meteor-node-stubs": "~0.2.0",
    "moment": "^2.18.1",
    "prop-types": "^15.5.10",
    "raven-js": "^3.15.0",
    "react": "^15.4.2",
    "react-apollo": "^1.0.0-rc.3",
    "react-burger-menu": "^2.0.2",
    "react-clipboard": "^1.3.0",
    "react-dom": "^15.4.2",
    "react-html-email": "^1.2.0",
    "react-redux": "^5.0.5",
    "react-router": "^2.5.2",
    "react-sidebar": "^2.3.2",
    "redux": "^3.6.0",
    "redux-devtools-extension": "^2.13.2",
    "redux-thunk": "^2.2.0",
    "subscriptions-transport-ws": "^0.5.5",
    "tracker-component": "^1.3.21"
  },
```

**Comment by aaronOSI** [comment]
Having this problem as well with 1.3.0 / React 15.3.2. I will try to make a PR but until then: https://stackoverflow.com/questions/38504879/error-cannot-find-module-react-lib-object-assign-from-dispatcher

**Comment by WickyNilliams** [comment]
Looks like I forgot to remove this line when switching to JSX: https://github.com/WickyNilliams/react-clipboard/blob/8548f5a7a6833bb97e4542f5a6fc81b76100b781/src/index.js#L3.

Will fix now

**Comment by WickyNilliams** [comment]
Fixed in `v1.3.1`, published to npm
